language-design.md

```diff
@@ -50,6 +71,11 @@ What does the post on grayscale tell us about the process of API design?
 
 **Response**
 
+    I think the post tells us how there can be multiple meanings for a concept/problem. For instance, some users will prefer the first option, others the second or third, even though they follow the same process but with different naming conventions. The only different implementation is the 4th option, which makes clear that "you will need to change the function name to get a semi-transparent version of the color. Also, if in the future one needs to change the color to not be a shade of gray, a function name change is not needed."[Verou, 2014].
+    I think this also connects to How to Design a Good API and Why It Matters as function names being changed ties back into being able to add in features but not being able to remove features.
+    What I'm trying to say is that the the  notation for grayscale colors can seem like a very insignificant design decision, but in the realm of API design this problem can have real impacts on the work people do, and it's the reasons outlined in Bloch's paper that this should be carefully navigated.
```

Comment:
This also connects to what you were talking about earlier, how it is important to get advice from other people when designing an API. Just because something makes sense to you, it could be confusing and annoying to another programmer. It is important to determine what the general consensus of the users of the language is.